packages/react-native-dogfood/src/screens/Call/CallScreen.tsx

```diff
@@ -33,6 +37,16 @@ export const CallScreen = ({ navigation }: Props) => {
     return () => subscription.unsubscribe();
   }, [answerCall, incomingCall]);
 
+  useEffect(() => {
+    if (!activeCall) {
+      return;
+    }
+    startForegroundService();
+    return () => {
+      stopForegroundService();
+    };
+  }, [activeCall]);
```

Comment:
These `useEffect` should become a hook to reduce duplication and to be used by our future integrators.

Something like `useForegroundService` or `useKeepCallAlive`

Reply:
Let's do this in the [useCallCycleEvents](https://github.com/GetStream/stream-video-js/blob/f11633d0a0f859ec579ffa670dbd2a8ba5227a44/packages/react-native-sdk/src/hooks/useCallCycleEffect.tsx#L23)